Sprint-2/errors/0.js

```diff
@@ -7,3 +7,8 @@ function capitalise(str) {
   let str = `${str[0].toUpperCase()}${str.slice(1)}`;
   return str;
 }
+
+/* In line 7, there is a declaration of the variable str but it is also in the scope of itself.
```

Comment:
We could remove the "let" because str is already an existing parameter in the function, it does not need to be redeclared. When we remove "let" it would then mean a modification of the existing variable to allow it to be returned when calling "return str"